### DISCUSSION TITLE
After successfully running apt-offline install command, is "synced" the proper message and the final step?

### DISCUSSION THREAD

**Discussion by Joel-Robertson** [rickysarraf/apt-offline#172] [Q&A] [answered]
Good day,

After running through all the steps successfully, the final step seems to be:

*On the offline machine*
apt-offline install [path_to_offline_data]/

This seems to work, and I see a whole list of ubuntu packages that are "synced". Is this the final step? I am a little confused if I need to run any other commands to actually install the downloaded/synced updates.

Thank you,

J

**Comment by rickysarraf** [comment, marked as answer]
After the last command for `apt-offline` that you've mentioned, you should continue with your usual `apt` installation commands. And `apt` will detect that there's no extra payload to download.

The wiki has this documented. I also prepared a video last year, demonstrating the use cases.

* https://github.com/rickysarraf/apt-offline/wiki
* https://www.youtube.com/watch?v=3Z7s-8vsWsw

**Reply by Joel-Robertson** [reply]
Excellent video. I see that apt should handle everything after the [apt-offline install] command.

For my case, I have multiple package updates downloaded and bundled together on the offline machine. I saw in the video that you used apt install [fish] to install a brand new single package. Should I run [apt install bundle.zip]? It looks like my apt is still trying to reach out to the internet instead of look internally.

Additionally, I checked in my /var/cache/apt/archives directory and there is no data present (only a blank lock file and an empty /partial folder). Should this directory have data populated in it?

J

**Reply by rickysarraf** [reply]
`apt` and `apt-offline` are different tools. `apt-offline` is an enabler for `apt` to bridge the network gap.

Please re-read the manual page and/or revisit the video I linked.

To install a package, the wholesome steps would be:

* apt-offline set /tmp/fish.uris --install-package fish
* apt-offline get /tmp/fish.uris --bundle /tmp/fish.zip
* apt-offline install /tmp/fish.zip
* apt install fish

**Reply by rickysarraf** [reply]
Also, if you think the manual page and/or the wiki documentation could be made clearer, I'd appreciate if you could contribute back in the form of a PR. :-)
